maro/simulator/scenarios/vm_scheduling/meta/source_urls.yml

```diff
@@ -1,9 +1,5 @@
 AzurePublicDataset:
 vm_data:
-  azure.2019.original:
-    remote_url: "https://raw.githubusercontent.com/Azure/AzurePublicDataset/master/AzurePublicDatasetLinksV2.txt"
-    sample: 2695548
-    seed: 9
   azure.2019.336k:
     remote_url: "https://raw.githubusercontent.com/Azure/AzurePublicDataset/master/AzurePublicDatasetLinksV2.txt"
```

Comment:
Not sure how to modify the structure. Since different topos should use different urls (logically)? In the future, we will add other public data, such as aliyun, google cluster, etc. They will use different urls.

Unless add one more key with sth. like "azure.2019" and extract from the user input "topology". 
And in the future, it might looks like 
```yaml
vm_data:
  azure.2019:  
    remote_url: "https://raw.githubusercontent.com/Azure/AzurePublicDataset/master/AzurePublicDatasetLinksV2.txt"
    [topology]:
      xx: xx
  azure.2017:
    remote_url: "xx"
    [topology]:
      xx: xx
  google.2019:
    remote_url: "xx"
    [topology]:
      xx: xx
  google.2017:
    remote_url: "xx"
    [topology]:
      xx: xx
```

Reply:
I noticed that when the data pipeline is generated, it will iterate vm_data's key. So we should put the topology in the second layer. It might could not use the structure as above.